core/templates/dev/head/pages/library/library.html

```diff
@@ -130,6 +130,7 @@ <h2 ng-class="{'active': activeGroupIndex === $index}" class="oppia-library-grou
                                              category="tile.category"
                                              thumbnail-icon-url="tile.thumbnail_icon_url"
                                              thumbnail-bg-color="tile.thumbnail_bg_color"
+                                             activity-is-owned-by-current-user="activitiesOwned.collections[tile.id]"
```

Comment:
what if the user is logged out? If I'm reading the code correctly, $scope.activitiesOwned is not defined in that case

Reply:
In that case the ownership is not defined and ng-if takes not defined as false, so that won't cause any error.

Reply:
I've declared the $scope.activitiesOwned variable before checking if the user is logged in. That way the variable is defined but if we try to access the tile.id , it'll give undefined and no errors